src/global-firestore-data/utils.ts

```diff
@@ -16,3 +16,6 @@ export const incrementUniqueID = (amount = 1): number => {
     .set({ uniqueIncrementer: updatedID });
   return updatedID;
 };
+
+export const isCourseConflict = (uniqueId: string | number): boolean =>
+  store.state.doubleCountedCourseUniqueIDSet.has(uniqueId);
```

Comment:
I don't think this is specific to `global-firestore-data` (which should be for functions directly accessing or manipulating the firestore data), maybe it should go in `requirement-frontend-utils`?

Reply:
Moving this to `requirement-frontend-utils` leads to a dependency cycle between that file and `store`. The `utils` file it is in now already has the `store` import so leaving it there might work, even though I agree it is not specific to `global-firestore-data`. Do you have any other suggestions?

Reply:
you can put it in `store.ts` (maybe a new file `store-utils.ts` or straight up not have it refactored. I think it's definitely more confusing where it is now